drivers/CoreVideo/include/CoreVideo.hpp

```diff
@@ -13,13 +12,14 @@
 #include "interface/LCD.hpp"
 #include "interface/LTDC.hpp"
 #include "interface/SDRAM.hpp"
+#include "interface/drivers/STM32Hal.h"
 
 namespace leka {
 
 class CoreVideo
 {
   public:
-	CoreVideo(CoreSTM32HalBase &hal, interface::SDRAM &coresdram, interface::DMA2DBase &coredma2d,
+	CoreVideo(interface::STM32Hal &hal, interface::SDRAM &coresdram, interface::DMA2DBase &coredma2d,
 			  interface::DSIBase &coredsi, interface::LTDCBase &coreltdc, interface::LCD &corelcd,
 			  interface::Graphics &coregraphics, interface::Font &corefont, interface::JPEGBase &corejpeg);
```

Comment:
Super, on les a tous ici!

```suggestion
	CoreVideo(interface::STM32Hal &hal, interface::SDRAM &coresdram, interface::DMA2D &coredma2d,
			  interface::DSI &coredsi, interface::LTDC &coreltdc, interface::LCD &corelcd,
			  interface::Graphics &coregraphics, interface::Font &corefont, interface::JPEG &corejpeg);
```